test/unit/specs/vislib/visualizations/tile_map.js

```diff
@@ -196,6 +196,41 @@ define(function (require) {
           });
         });
       });
+
+      describe('getMinMax method', function () {
+        it('should return an object', function () {
+          vis.handler.charts.forEach(function (chart) {
+            var data = chart.handler.data.data;
+            expect(_.isObject(chart.getMinMax(data))).to.be(true);
```

Comment:
You don't need lodash here, use `            expect(chart.getMinMax(data)).to.be.an(Object);
` instead